libs/community/tests/integration_tests/vectorstores/test_infinispanvs.py

```diff
@@ -1,7 +1,9 @@
 """Test Infinispan functionality."""
```

Comment:
FYI: There's a standard testing suite for vectorstores in langchain_standard_tests, it'll likely weed out a number of bugs